src/input-group.scss

```diff
@@ -201,6 +201,31 @@ $fd-input-border-radius: var(--sapField_BorderCornerRadius);
 
     border: none;
     width: 100%;
+
+    &.is-expanded {
+      background-color: var(--sapButton_Active_Background);
+      color: var(--sapButton_Active_TextColor);
+    }
+  }
+
+  &--control {
+    @include fd-hover() {
+      .#{$block}__button {
+        background-color: var(--sapButton_Lite_Hover_Background);
+
+        &.is-expanded {
+          background-color: var(--sapButton_Active_Background);
```

Comment:
Shouldn't this be refactored to use SCSS variables initialized with CSS variables or values at the top of this selector?  (Same for the rest of edited components)

Reply:
Done